src/Modules/Authontication/Components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,170 @@
-import React from 'react'
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import { PasswordValidation } from "../../../../constans/VALIDATIONS";
+import axios from "axios";
+import { AuthorizedToken, USERS_URLs } from "../../../../constans/END_POINTS";
+import { toast } from "react-toastify";
 
 export default function ChangePass() {
+  const [showOldPassword, setShowOldPassword] = useState(false);
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm({ mode: "onBlur" });
+
+  const toggleVisibility = (setterFunction: any) => {
+    setterFunction((prevState: any) => !prevState);
+  };
+
+  const onSubmit =  async(data: any) => {
+    try {
+      const response: any =await axios.put(USERS_URLs.ChangePassword, AuthorizedToken, data);
```

Comment:
We can provide the type of the response to the put method  using generics like this => `axios.put<{data:{message:"string"}}>`. We just need to look at the structure coming from the response and add it. This will ensure that we are accessing the right properties in the response object and detect any potential errors.